hangman/test/hangman_test.exs

```diff
@@ -0,0 +1,155 @@
+defmodule HangmanTest do
```

Comment:
### Fantastic set of tests.  +8

Just a suggestion: you could have made your life simpler by adding a new function to Hangman:

~~~ elixir
def new_word() do
  Dictionary.random_word()
  |> new_game()
end

def new_game(word) do 
 # ...
end
~~~